### DISCUSSION TITLE
怎样配置 Aliyun Office Previewer 作为文档预览？

### DISCUSSION THREAD

**Discussion by Mr-LiuDC** [AlistGo/alist#3072] [General]
运行方式 Docker，xhofe/alist:latest
版本：3.8.0
系统：CentOS 7

我看到某些别人的Alist配置的 Aliyun Office Previewer 作为默认的文档预览工具，但是我的没有，还是默认的 Microsoft和Google，而且这两个还受网络影响严重，请问该怎样配置 Aliyun Office Previewer 作为文档预览工具？

别人的例子：https://alist.yyej.com/haidaoit/%E5%85%B6%E5%AE%83/%E5%84%BF%E7%AB%A5%E7%BB%98%E6%9C%AC%E5%A4%A7%E5%85%A8%EF%BC%88%E8%BF%91%E5%8D%83%E6%9C%AC%EF%BC%8C3.4G%EF%BC%89/%E5%A4%A7%E9%97%AE%E9%A2%98.pptx

我的例子：https://netdisk.decai.pub/Office_Document/Example_PowerPoint.pptx

**Comment by anwen-anyi** [comment]
只有阿里的才可以默认使用阿里的
其他的只有微软和谷歌，或者自己搭建一个onlyoffice

**Reply by Mr-LiuDC** [reply]
我现在挂载了一个阿里云盘的分享链接，提示 Not Implement
而且我还发现微软和谷歌的文档预览，档文件放在OSS里面的时候预览正常，档在服务器目录挂载的时候就不能预览。

![image](https://user-images.githubusercontent.com/18117854/212880199-f60ed28f-8ef0-43de-9a35-408d39843bb0.png)

**Comment by NovaShen555** [comment]
https://help.aliyun.com/document_detail/445755.html
若仍报错请再回复

**Comment by zhushen12580** [comment]
> 运行方式 Docker，xhofe/alist:latest 版本：3.8.0 系统：CentOS 7
> 
> 我看到某些别人的Alist配置的 Aliyun Office Previewer 作为默认的文档预览工具，但是我的没有，还是默认的 Microsoft和Google，而且这两个还受网络影响严重，请问该怎样配置 Aliyun Office Previewer 作为文档预览工具？
> 
> 别人的例子：https://alist.yyej.com/haidaoit/%E5%85%B6%E5%AE%83/%E5%84%BF%E7%AB%A5%E7%BB%98%E6%9C%AC%E5%A4%A7%E5%85%A8%EF%BC%88%E8%BF%91%E5%8D%83%E6%9C%AC%EF%BC%8C3.4G%EF%BC%89/%E5%A4%A7%E9%97%AE%E9%A2%98.pptx
> 
> 我的例子：https://netdisk.decai.pub/Office_Document/Example_PowerPoint.pptx

大佬，请问下您的搜索功能怎么做的呀？

**Reply by Mr-LiuDC** [reply]
没做特殊设置吖，是不是你的版本太低了。
